pkg/apps/utils.tsx

```diff
@@ -142,3 +170,7 @@ export const launch = (comp: Component) => {
         }
     }
 };
+
+export function reload_bridge_packages() {
+    return cockpit.dbus(null, { bus: "internal" }).call("/packages", "cockpit.Packages", "Reload", []);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22611-83e71f7c-20251121-122204-fedora-43-devel/Coverage/lcov/github-pr.diff.gcov.html)